x-pack/plugins/canvas/public/components/fullscreen/fullscreen.scss

```diff
@@ -1,5 +1,5 @@
 body.canvas-isFullscreen { // sass-lint:disable-line no-qualifying-elements
-  // following two rules are for overriding the header bar padding 
+  // following two rules are for overriding the header bar padding
```

Comment:
I think this might need to be adjusted because canvas in fullscreen is having some issues:
<img width="1144" alt="_eCommerce__Revenue_Tracking_-_Kibana_and_Stacked_headers_and_navigational_search_by_myasonik_·_Pull_Request__72331_·_elastic_kibana" src="https://user-images.githubusercontent.com/766596/92264976-b95abc00-eea4-11ea-96aa-fc249acbc5c8.png">